spec/ace.md

```diff
@@ -131,7 +131,75 @@ An executor MAY implement a "strict mode" where an image cannot run unless all i
 ### Linux Isolators
 
 These isolators are specific to the Linux kernel and are impossible to represent as a 1-to-1 mapping on other kernels.
-The first example is "capabilities" but this will be expanded to include things such as SELinux, SMACK or AppArmor.
+This set includes Linux-specific isolators, which may relies on kernel technologies like seccomp, SELinux, SMACK or AppArmor.
+
+#### os/linux/seccomp-remove-set
+
+* Scope: app
+
+**Parameters:**
+
+* **set** case-sensitive list of syscall names that will be blacklisted (ie. blocked); values starting with `@` MUST be handled as scoped special values (see notes below). This field MUST NOT be empty. All syscalls specified in this set MUST be blocked. This set MAY be augmented by an implementation-specific default blacklist set (see notes below). Syscalls not specified in the union of these two sets MUST NOT be blocked.
+* **errno** all-uppercase name of a single [errno code](http://man7.org/linux/man-pages/man3/errno.3.html) that will be returned by blocked system calls, instead of terminating. If missing or empty, by default blocked syscalls MUST result in app termination via `SIGSYS` signal.
```

Comment:
Moving this to a MUST for POSIX.1 and C99 errnos, and a SHOULD for all the others. This will account for future entries, whenever Linux introduces new custom errno (as the list may be kernel-version dependent).